### PR TITLE
Continue Tabs work (Rel #10424)

### PR DESCRIPTION
**Pull request by rcantin-w**
## Who is this for?
Devs/continuity of #10424, just thought it'd be good to see the changes from David's work as separate.
Relates to #10239 

## What is it doing for them?
- Here I'm suggesting a new property (replacing `isLinks`): `behaviourVariant` where we are forced to declare a behaviour on ALL Tabs, as its two valid types are `anchor` or `tab`. I believe this does two things:
  1. Improves readability when coming across a Tabs component, you know exactly what it does.
  2. Enforces type check. Once you've declared a `behaviourVariant`, it expects certain properties and will error if given the wrong ones.
- Fixes e2es as selectors had changed (not a button anymore)
- Renamed the sub-components to match the `behaviourVariant` values, but I know this was discussed and decided on so do let me know if you'd I revert and change the `behaviourVariant` values instead (@gestchild)
- Worked on focus state + tweak colours and spacing (agreed with @j-jaworski )
- Add Readme to stories and add some customisation options
- Align some of the shared properties between Tab and Anchor (id/label, isWhite)

### CODE REVIEW

File: common/views/themes/base/fonts.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const fonts = css<GlobalStyleProps>`
   @font-face {
     font-family: 'Inter';
     font-style: normal;
-    font-weight: 400;
+    font-weight: 500;
```
Comment (rcantin-w) [inline common/views/themes/base/fonts.ts:8]: Had a chat with Jason and "normal" would usually be 500, lmk if it was wrong to change.

File: content/webapp/components/Tabs/Tabs.Tab.tsx [hunk 2]
```diff
@@ -43,18 +43,17 @@ type SelectableTextLink = {
...
   items: SelectableTextLink[];
   selectedTab: string;
   setSelectedTab: Dispatch<SetStateAction<string>>;
   isWhite?: boolean;
   trackWithSegment?: boolean;
 };
 
-const Tabs: FunctionComponent<Props> = ({
-  id,
+const TabbableTabs: FunctionComponent<Props> = ({
```
[the remark this answers is not in the document]
Comment (rcantin-w) [reply content/webapp/components/Tabs/Tabs.Tab.tsx:55]: Could go back to `default` for this one?

[the remark this answers is not in the document]
Comment (gestchild) [reply content/webapp/components/Tabs/Tabs.Tab.tsx:55]: I can't think of anything better

**Comment by gestchild** [comment]
I noticed an issue with the tabs when javascript is disabled. Probably best to keep it separate from this PR, so [I've created an issue](https://github.com/wellcomecollection/wellcomecollection.org/issues/10437)

File: content/webapp/components/Tabs/Tabs.Tab.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (gestchild) [reply content/webapp/components/Tabs/Tabs.Tab.tsx:55]: do you mean back to 'Tabs', i prefer that to default

[the remark this answers is not in the document]
Comment (gestchild) [reply content/webapp/components/Tabs/Tabs.Tab.tsx:55]: Or maybe something like TabList because that is the aria-role

**Review by gestchild — APPROVED** [review]

File: content/webapp/components/Tabs/Tabs.Tab.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (rcantin-w) [reply content/webapp/components/Tabs/Tabs.Tab.tsx:55]: I could do `Tabs` vs `Tabs.Anchor`, as if the latter was the alternative. We're talking around removing the tablist role right?

[the remark this answers is not in the document]
Comment (gestchild) [reply content/webapp/components/Tabs/Tabs.Tab.tsx:55]: we are yeah, and thinking about it they are actually all anchors in that case. The 'Tabs' ones are in page anchors, but they're default behaviour will get overwritten with javascript

[the remark this answers is not in the document]
Comment (rcantin-w) [reply content/webapp/components/Tabs/Tabs.Tab.tsx:55]: Ah right... so maybe the name should talk to the behaviour the user sees.
`TabsSwitch` and `TabsNavigate` for the type : `"navigate" | "switch"`? Navigates to content vs switches the content.
I don't love it but am blanking.

[the remark this answers is not in the document]
Comment (gestchild) [reply content/webapp/components/Tabs/Tabs.Tab.tsx:55]: I prefer that actually

[the remark this answers is not in the document]
Comment (rcantin-w) [reply content/webapp/components/Tabs/Tabs.Tab.tsx:55]: Alright let's go with that for now. Thank you!

**Pull request merged**